Description: Simplify imports
 Importing a whole module using a wildcard is frowned upon, and it
 confuses static analysis tools such as pyflakes.
Author: Andrea Bolognani <eof@kiyuko.org>
Forwarded: Marv Boyes <greyspace@tuxfamily.org>
Last-Update: 2012-01-30
Index: rhinote.git/rhinote.py
===================================================================
--- rhinote.git.orig/rhinote.py
+++ rhinote.git/rhinote.py
@@ -21,14 +21,13 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St., Fifth Floor, Boston, MA  02110-1301 USA
 
-from Tkinter import *
+import Tkinter
 import tkFileDialog, tkMessageBox
 import os
-from os import system
 
 # the root window:
 def Rhinote():
-    r = Tk()
+    r = Tkinter.Tk()
     r.option_add('*font', '{Helvetica} 11')
     t = TextWidget(r, bg = '#f9f3a9', wrap = 'word', undo = True)
     t.focus_set()
@@ -38,7 +37,7 @@ def Rhinote():
     r.mainloop()
 
 # the text widget, and all of its functions:
-class TextWidget(Text):
+class TextWidget(Tkinter.Text):
 
     def save_file(self, whatever = None):
         if (self.filename == ''):
@@ -86,7 +85,7 @@ class TextWidget(Text):
         f.close
         # 'enscript' formats the text; lpr sends it to the default printer;
         # enscript's -B option suppresses page headers.
-        system('enscript -B --word-wrap $HOME/.Rhinoteprintfile > lpr &')
+        os.system('enscript -B --word-wrap $HOME/.Rhinoteprintfile > lpr &')
 
     def help(self, whatever = None):
         tkMessageBox.showinfo('Rhinote Help', message = '''
@@ -113,7 +112,7 @@ http://rhinote.tuxfamily.org
 ''')
 
     def __init__(self, master, **kw):
-        Text.__init__(self, master, **kw)
+        Tkinter.Text.__init__(self, master, **kw)
         self.bind('<Control-n>', self.new_window)
         self.bind('<Control-N>', self.new_window)
         self.bind('<Control-o>', self.open_file)
